flycheck.el

```diff
@@ -6182,6 +6183,43 @@ Requires DMD 2.066 or newer.  See URL `http://dlang.org/'."
             (or "Warning" "Deprecation") ": " (message) line-end))
   :modes d-mode)
 
+(defun flycheck-parse-elixir-dogma(output checker buffer)
+  "Parse dogma errors from JSON OUTPUT.
+
+CHECKER and BUFFER denoted the CHECKER that returned OUTPUT and
+the BUFFER that was checked respectively.
+
+See URL `https://github.com/lpil/dogma' for more information
+about dogma."
+  (cl-flet ((alist-get (key alis) (cdr (assoc key alis))))
```

Comment:
Besides I think you can use assq instead of assoc here as all the keys are symbols.